native/package.json

```diff
@@ -120,10 +120,12 @@
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
     "metro-react-native-babel-preset": "^0.59.0",
+    "react-native-flipper": "^0.62.0",
     "react-native-git-upgrade": "^0.2.7",
     "react-native-svg-asset-plugin": "^0.4.0",
     "react-test-renderer": "16.13.1",
     "redux-devtools-extension": "^2.13.8",
```

Comment:
We should probably remvoe react-devtools-extension when switching integrating flipper.

Reply:
Should I remove it or will this be done in a separate task?

Reply:
If the changes with flipper are done in this issue, it should be removed. If you'll move the flipper stuff to a separate task, it should be done there I'd say.

Reply:
I'm not sure what changes exactly have to be made for the flipper integration. At the current state of my PR everything works for me, but I don't know if there are any things missing.

Reply:
Then its probably the best to do this in a separate task. We probably have to check if enabling hermes breaks something as well.

Reply:
Alright. Then I'll also disable hermes before merging. Leaving the redux-flipper and react-native-flipper packages shouldn't harm anything though.

Reply:
In that case create a task for flipper and mention the changes which have been done already.

Reply:
There is already a task: https://issues.tuerantuer.org/browse/IGAPP-334